nss-day-cohort-45/holiday-road-cold-shoulders
# Weather Forecast MVP

## Changes:

Fixed bug that was causing days to display strangely when a user clicks more than one park.
Fixed bug that was causing Saturday to become all the days instead of resetting.
Added a placeholder logo to make me laugh while testing.

## To Test:

Weather is no longer making use of the JSON file, and instead uses the Open Weather API. If you're testing a lot on your end you may want to comment out line 46 through 58 on weatherList.js.

Make sure the weather is displaying properly when choosing a park. Then choose another park to check that it populates properly no matter how many parks the user chooses.
